index.js

```diff
@@ -4,27 +4,23 @@ module.exports = function (img, selection_canvas, onCrop) {
   if('function' === typeof selection_canvas)
     onCrop = selection_canvas, selection_canvas = null
 
-  var c2 = selection_canvas = selection_canvas || h('canvas.hypercrop__selection', {width: 512, height: 512})
-
   onCrop = onCrop || function () {}
 
- var width = img.width, height = img.height
+  var c2 = selection_canvas = selection_canvas || h('canvas.hypercrop__selection', {width: 512, height: 512})
+
+  var width = img.width
+  var height = img.height
 
   var c = CANVAS = h('canvas.hypercrop__canvas', {
```

Comment:
Sorry, this is a debugging artifact... add something as a global to make it accessible in the console. When I do this, I always do it in CAPS.